internal/cmd/asyncapi/command_asyncapi_test_utils.go

```diff
@@ -0,0 +1,228 @@
+package asyncapi
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	ccloudv1 "github.com/confluentinc/ccloud-sdk-go-v1-public"
+	ccloudv1mock "github.com/confluentinc/ccloud-sdk-go-v1-public/mock"
+	kafkarestv3 "github.com/confluentinc/ccloud-sdk-go-v2/kafkarest/v3"
+	kafkarestv3mock "github.com/confluentinc/ccloud-sdk-go-v2/kafkarest/v3/mock"
+	srsdk "github.com/confluentinc/schema-registry-sdk-go"
+	srMock "github.com/confluentinc/schema-registry-sdk-go/mock"
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/cli/internal/pkg/ccloudv2"
+	"github.com/confluentinc/cli/internal/pkg/ccstructs"
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/config"
+	v1 "github.com/confluentinc/cli/internal/pkg/config/v1"
+	dynamicconfig "github.com/confluentinc/cli/internal/pkg/dynamic-config"
+	"github.com/confluentinc/cli/internal/pkg/version"
+	testserver "github.com/confluentinc/cli/test/test-server"
+)
+
+const BackwardCompatibilityLevel = "BACKWARD"
+
+var details = &accountDetails{
+	cluster: &ccstructs.KafkaCluster{
+		Id:        "lkc-asyncapi",
+		Name:      "AsyncAPI Cluster",
+		Endpoint:  "http://kafka-endpoint",
+		AccountId: "env-asyncapi",
+	},
+	srClient: &srsdk.APIClient{
```

Comment:
Can you use the integration testing framework in `test/` instead of mocking the Schema Registry backend in a single variable?